scenes/SceneSlate.js

```diff
@@ -302,6 +306,33 @@ class SlatePage extends React.Component {
     }, 1000);
   };
 
+  _handleURLRedirect = () => {
+    const {
+      page: { cid },
+    } = this.props;
+
+    /* NOTE(daniel): If user was redirected to this page, the cid of the slate object will exist in the page props. 
+    We'll use the cid to open the global carousel */
+    if (cid) {
```

Comment:
Can you change this to

`if (Strings.isEmpty(cid)) { return null }`

And then the bottom code doesn't have to be in a `{}` if closure.